### PR TITLE
RS Virtualization docs and yaml

### PR DESCRIPTION
**Pull request by raagupta**
Added docs, yaml files and images related to RS Virtuaslization

### CODE REVIEW

File: docs/rs-virtualization/installation-steps.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+## Step 5: Deploy Grafana Dashboard
+[Here](../../data-assets/rs-virtualization/deploy/vm_right_sizing_grafana_dashboard.yaml) is the yaml file that contains configuration of ACM Right-Sizing grafana dashboard. Use the following command to add this dashboard on Grafana. 
+
+```
+oc create -f data-assets/rs-virtualization/deploy/vm_right_sizing_grafana_dashboard.yaml
+```
+
+## Step 6: Access Grafana Dashboard
+Wait for some time for the Prometheus Recording Rules for Vistualization to get triggered, and we will have some data points. You can then go to Grafana and search for the `ACM Right-Sizing Namespace` dashboard. Click on it, and you are ready to use Right Sizing for Virtualization solution.
```
Comment (rajusem) [inline docs/rs-virtualization/installation-steps.md]: Please correct the name of the dashboard

File: data-assets/rs-virtualization/deploy/rs-vm-rules-policy.yaml [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+                  namespace: openshift-monitoring
+                spec:
+                  groups:
+                    - interval: 5m
+                      name: acm-vm-right-sizing-namespace-5m.rule
+                      rules:
+                        - expr: |
+                            max_over_time(
+                              sum((
+                                    kubevirt_vm_resource_requests{
```
Comment (rajusem) [inline data-assets/rs-virtualization/deploy/rs-vm-rules-policy.yaml:36]: Can we update doc and add some high level info on metric used to calculate cpu/memory usage/request

**Comment by rajusem** [comment]
lgtm

**Review by dvandra — APPROVED** [review]
LGTM

**Pull request merged**